src/components/tabs/js/tabScroll.js

```diff
@@ -0,0 +1,22 @@
+(function () {
+  'use strict';
+  angular.module('material.components.tabs')
+      .directive('mdTabScroll', MdTabScroll);
+
+  function MdTabScroll () {
```

Comment:
There's no angular directive for mousewheel to my knowledge.  I wrote this to be a generic re-usable mousewheel directive.  I named it mdTabScroll for now, but it could easily be reused in other components.